src/plugins/textReplace.tsx

```diff
@@ -0,0 +1,274 @@
+                <TextReplaceRegex />
+            </>
+    },
+});
+
+const TextReplaceString = () => {
+    const update = useForceUpdater();
+
+    async function onClickRemoveString(index: number) {
+        if (index === rulesString.length - 1) return;
+        rulesString.splice(index, 1);
+        await DataStore.set("TextReplace_rulesString", rulesString);
+        update();
+    }
+
+    async function onChangeString(e: string, index: number, key: string) {
+        if (index === rulesString.length - 1) {
+            rulesString.push({
+                find: "",
+                replace: "",
+                onlyIfIncludes: ""
+            });
+        }
+        rulesString[index][key] = e;
+        if (rulesString[index].find === "" && rulesString[index].replace === "" && rulesString[index].onlyIfIncludes === "" && index !== rulesString.length - 1) {
+            rulesString.splice(index, 1);
+        }
+        await DataStore.set("TextReplace_rulesString", rulesString);
+        update();
+    }
+
+    return (
+        <>
+            <h2 style={{
+                color: "#fff"
+            }}>Using string</h2>
+            <table>
+                {
+                    rulesString?.map((rule: any, index: number) =>
+                        <tr>
+                            <td>
+                                <TextInput
+                                    placeholder="Find"
+                                    value={rule.find}
+                                    onChange={e => onChangeString(e, index, "find")}
+                                    spellCheck={false}
+                                />
+                            </td>
+                            <td>
+                                <TextInput
+                                    placeholder="Replace"
+                                    value={rule.replace}
+                                    onChange={e => onChangeString(e, index, "replace")}
+                                    spellCheck={false}
+                                />
+                            </td>
+                            <td>
+                                <TextInput
+                                    placeholder="Only if includes"
+                                    value={rule.onlyIfIncludes}
+                                    onChange={e => onChangeString(e, index, "onlyIfIncludes")}
+                                    spellCheck={false}
+                                />
+                            </td>
+                            <Button
+                                size={Button.Sizes.MIN}
+                                onClick={() => onClickRemoveString(index)}
+                                style={{
+                                    background: "none",
+                                    bottom: "-7px"
+                                }}
+                            >
+                                <svg width="24" height="24" viewBox="0 0 24 24">
```

Comment:
Currently people using a screenreader cannot tell what this button does, you should add a title to the svg 
```suggestion
                                <svg width="24" height="24" viewBox="0 0 24 24">
                                	<title>Delete Rule</title>
```